use-location.js

```diff
@@ -75,5 +75,5 @@ const patchHistoryEvents = () => {
   return (patched = 1);
 };
 
-const currentPathname = (base, path = location.pathname) =>
-  !path.indexOf(base) ? path.slice(base.length) || "/" : path;
+const currPathname = (base, path = location.pathname.toLowerCase()) =>
```

Comment:
Actually, it feels like the path doesn't have to be lowercased. The matcher will take care of it (you can test it as weel).
```suggestion
const currPathname = (base, path)) =>
```

Reply:
oh~,nice, can reduce some bytes, i will test it

Reply:
`location.pathname` should be lowercased, so we can slice the `location.pathname` with `base` successfully.
e.g.

```js
location.pathname = '/App/foo'
// and
base = '/app'
```

Reply:
Oh, you're right, my bad 😄